backend/src/main/java/gov/cdc/usds/simplereport/config/simplereport/SmartyStreetsConfig.java

```diff
@@ -0,0 +1,25 @@
+package gov.cdc.usds.simplereport.config.simplereport;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConstructorBinding;
+
+@ConfigurationProperties(prefix = "simple-report.address-validation")
```

Comment:
since this is more coupled to SmartyStreets than it is to address validation, can you change this config path to simple-report.smarty-streets? or just smarty-streets with no preix? then the variable names become authId and authToken, etc